mozilla_django_oidc/urls.py

```diff
@@ -16,3 +18,16 @@
         name='oidc_authentication_init'),
     url(r'^logout/$', views.OIDCLogoutView.as_view(), name='oidc_logout'),
 ]
+
+
+if 'test' in sys.argv:
+    # If we're running tests, it helps to have a "home page" we can test
+    # against that doesn't trigger authentication things.
+    from django.http import HttpResponse
+
+    def fakeview(req):
+        return HttpResponse('Win!')
+
+    urlpatterns.append(
+        url(r'^mdo_fake_view/$', fakeview, name='mdo_fake_view')
+    )
```

Comment:
I don't know a better way of doing this where "better" doesn't involve creating a django project in the tests or something equally large and finicky across Django versions.

I'm pretty sure the `if 'test' in sys.argv` will work, but if there's a better way to determine we're in a test run, I'm all ears.